contracts/test/PermitSigner.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.15;
+
+import "./SigUtils.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/extensions/draft-IERC20PermitUpgradeable.sol";
+
+contract PermitSigner {
+  function digestPermitCall(
```

Comment:
I was originally trying to have this call `vm.sign` in this function but the import of foundry std lib confuses the hardhat compilation phase...